app/uk/gov/hmrc/cgtpropertydisposalsfrontend/views/components/unit_input_govuk.scala.html

```diff
@@ -103,7 +103,6 @@ <h1 class="govuk-label-wrapper">
               id="@fieldId"
               name="@fieldId"
               type="text"
-              inputmode="decimal"
```

Comment:
will this input component work for decimal value without this attribute?

Reply:
`decimal` has issues with some devices, so it is best not to set an `inputmode` at all
See https://design-system.service.gov.uk/components/text-input/#asking-for-decimal-numbers